lib/jnpr/junos/device.py

```diff
@@ -1058,6 +1058,16 @@ def __new__(cls, *args, **kwargs):
         if kwargs.get('port') in [23, '23'] or kwargs.get('mode') or \
                         kwargs.get('cs_user') is not None:
             from jnpr.junos.console import Console
+            if kwargs.get('conn_open_timeout', None):
+                # Console already supports timeout while opening connections
+                # via `timeout` parameter. Refer `Console` documentation
+                # for more details.
+
+                # Note: The actual timeout may appear higher than set timeout as
+                # Console retries the connection attempt for 3 times, with a backoff
+                # time of 2s. SO, if my math is correct, the actual timeout is
```

Comment:
remove this `if my math is correct,`
we can just say `the actual timeout can be ....`

Reply:
Corrected.